pkg/backend/refresh.go

```diff
@@ -2,11 +2,29 @@ package backend
 
 import (
 	"context"
+	"time"
 
+	"github.com/hashicorp/vault/sdk/framework"
 	"github.com/hashicorp/vault/sdk/logical"
 	"golang.org/x/oauth2"
 )
 
+func tokenOk2Reuse(tok *oauth2.Token, data *framework.FieldData) bool {
```

Comment:
It took me a second to understand why this changed the reusability of this token vs. simply `tok.Valid()`. Maybe a name like `tokenValidWithClientRequirements` or something would make it more clear?

Reply:
That's an awfully long name.  I think if you don't like `tokenOk2Reuse` I'd rather just call it `tokenValid` or possibly `tokenValidForReuse`.  How about one of those?

Reply:
I changed it to `tokenValid()`.